crates/store/re_chunk/src/chunk.rs

```diff
@@ -73,6 +73,18 @@ impl ChunkComponents {
             .insert(component_desc, list_array);
     }
 
+    /// Returns the all list arrays for the given component name.
+    #[inline]
+    pub fn get_by_component_name<'a>(
```

Comment:
Casually introducing wildcard semantics makes me a bit uneasy :grimacing: 

But whatever, we'll have to redesign everything when introducing descriptor-driven queries anyway.

Reply:
I first went with returning first and saying is UDB for more, but I found this more elegant 🤷